app/src/main/java/org/stepic/droid/code/ui/CodeAnalyzer.kt

```diff
@@ -141,4 +146,21 @@ object CodeAnalyzer {
     }
 
     fun getBracketsPair(bracket: String) = brackets.entries.find { it.key == bracket || it.value == bracket }
+
+
+    fun resolveAutocomplete(cursorPosition: Int, lang: String, text: String): AutocompleteState {
+        val next = getNextSymbol(cursorPosition, text)
```

Comment:
`getNextSymbol ` is actually return string, so it is good to change return type to `Char` or rename to `getNextSybmolAsString`